maven-java/pom.xml

```diff
@@ -31,6 +31,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.release>8</maven.compiler.release>
```

Comment:
Not clear if we should keep source and target. I didn't find clear documentation about it. I may be that `release` is sufficient. In any case, I don't want to fiddle with this right now.